bot/cogs/snakes.py

```diff
@@ -15,6 +30,62 @@ class Snakes:
     def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
+    PYTHON_INFO = {
+        'name': 'Python',
+        'scientific-name': 'Pseudo anguis',
+        'image-url': 'https://www.python.org/static/community_logos/python-logo-master-v3-TM.png',
+        'url': 'https://en.wikipedia.org/wiki/Python_(programming_language)',
+        'map-url': 'https://ih0.redbubble.net/image.80621508.8934/flat,800x800,075,t.u1.jpg',
+        'description': 'Python is an interpreted high-level programming language '
+                       'for general-purpose programming. '
+                       'Created by Guido van Rossum and first released in 1991, '
+                       'Python has a design philosophy that emphasizes code readability, '
+                       'notably using significant whitespace.'
+    }
+
+    async def on_ready(self):
+        self.session = ClientSession(loop=self.bot.loop)
+        self.info_url = 'https://snake-facts.weebly.com/'
+        log.info('Session created.')
+
+        with open('./snakes.txt', encoding='utf-8') as f:
+            self.sneks = f.read().split('\n')
+            for i, snek in enumerate(self.sneks):
+                self.sneks[i] = snek.replace('\u200b', '').replace('\ufeff', '')
+        log.info('Snakes loaded.')
+
+    def no_sneks_found(self, name):
+        '''Helper function if the snake was not found in the directory.'''
+        em = discord.Embed(
+            title='No snake found.',
+            color=discord.Color.green()
+        )
+
+        snakes = get_close_matches(name, self.sneks)
+
+        if snakes:
+            em.description = 'Did you mean...\n'
+            em.description += '\n'.join(f'`{x}`' for x in snakes)
+        else:
+            snakes = 'https://github.com/SharpBit/code-jam-1/blob/master/snakes.txt'
+            em.description = f'No close matches found. Click [here]({snakes}) for the list of available snakes.'
+
+        return em
+
+    def format_info(self, data, color=discord.Color.green()):
+        '''Formats the info with the given data.'''
```

Comment:
`"""` are preferred in docstrings over `'''`

Reply:
Not required, but some documentation for the parameters would be nice, maybe some type annotations at the least to say what `data` should be.